packages/fast-components-styles-msft/src/slider/index.ts

```diff
@@ -70,18 +70,33 @@ const styles: ComponentStyles<SliderClassNameContract, DesignSystem> = {
         "&:active": {
             background: neutralForegroundActive,
         },
+        "@media (-ms-high-contrast:active)": {
```

Comment:
should this specify black-on-white or white-on-black? can you provide screenshots for both?

Reply:
when you specify `-ms-high-contrast:active` it covers all high contrast styling rules, including black-on-white and white-on-black